assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -194,14 +210,63 @@ const ChecklistTab = (props) => {
     [checklist]
   );
 
+  const selectElement = useCallback(
+    ({ elementId, elements, pageId }) => {
+      if (pageId) {
+        setCurrentPage({ pageId });
+      }
+      if (Array.isArray(elements)) {
+        setSelectedElementsById({
+          elementIds: elements.map(({ id }) => id),
+        });
+      } else if (elementId) {
+        setSelectedElementsById({ elementIds: [elementId] });
+      }
+    },
+    [setCurrentPage, setSelectedElementsById]
+  );
+
+  const getOnClick = useCallback(
+    ({ elementId, pageId, elements }) => {
+      if (!elementId && !pageId && !elements) {
+        return undefined;
+      }
+      return () => selectElement({ elementId, pageId, elements });
+    },
+    [selectElement]
+  );
+
+  const getHandleKeyPress = useCallback(
+    ({ elementId, elements, pageId }) => (event) => {
+      if (event.key === 'Enter') {
+        selectElement({ elementId, elements, pageId });
+      }
+    },
+    [selectElement]
+  );
+
   const renderRow = useCallback(
-    ({ message, help, id, pageGroup }) => (
-      <Row key={`guidance-${id}`} pageGroup={pageGroup}>
-        {message}
-        <HelperText>{help}</HelperText>
-      </Row>
-    ),
-    []
+    ({ message, help, id, pageGroup, ...args }) => {
+      const onClick = getOnClick(args);
+      const handleKeyPress = getHandleKeyPress(args);
+      return (
+        <Row
+          tabIndex={0}
+          onClick={onClick}
+          // note: onKeyDown does not work for "Enter"
```

Comment:
Are you sure? I see some existing cases of `onKeyDown` in the code base that handle "Enter". For example in the `GoogleAnalyticsSettings` component.

Reply:
Yes, I'm sure :) I wish I had a better answer. I thought it might be my weird keyboard, but `enter` was the only key that didn't fire an event on this div.

Reply:
That's weird 🤔

Reply:
It showed up on Zach's page layout divs, as well! It's pretty strange. It detects the tabs, and all the other keys! It also detected `option+return` which would read as `Enter` but not just `Enter` alone...

Reply:
@barklund does this perhaps ring a bell to you?